frontend/static/js/pong/Player.js

```diff
@@ -0,0 +1,21 @@
+import * as THREE from 'three';
+
+class Player {
+	constructor(constants, isLeft) {
+		this.isLeft = isLeft;
+		this.player = this.createPlayer(constants);
+	}
+
+	createPlayer({players, game}) {
+		const playerGeometry = new THREE.BoxGeometry(players.width, players.height, 0);
+		const playerMaterial = new THREE.MeshBasicMaterial({color: 0xffffff});
+		const player = new THREE.Mesh(playerGeometry, playerMaterial);
+		if (this.isLeft) {
+			player.position.set(game.margin + players.width, game.height / 2, 0);
+		} else {
+			player.position.set(game.width - game.margin - players.width, game.height / 2, 0);
```

Comment:
do we get these measurements from the backend? or are we / should we make sure they match with the data from the backend or does the backend just send the middle coordinate of the player / ball?

Reply:
the initial positions are just the center of the screen for the ball (that's in the next PR fixed), and for the players on the y-axis center. But even so, as the backend immediately after sends positions, those are corrected, if they don't already agree already